core/model.py

```diff
@@ -46,20 +47,10 @@ def _predict(self, x):
         # this model does not like punctuation touching characters
         x = re.sub('([.,!?()])', r' \1 ', x)  # https://stackoverflow.com/a/3645946/
 
-        os.system("python lm_1b/lm_1b_eval.py --mode sample \
-                                     --prefix \"" + x + "\" \
-                                     --pbtxt assets/data/graph-2016-09-10.pbtxt \
-                                     --vocab_file assets/data/vocab-2016-09-10.txt  \
-                                     --ckpt 'assets/data/ckpt-*' \
-                                     --num_samples 1")
-
         try:
-            # if the above os.system command fails then "out.txt" is never created
-            txt_file = open("out.txt", "r")
+            output = lm_1b_eval.generate(x)
         except OSError:
```

Comment:
If it is about insufficient memory, `MemoryError` will be raised instead of `OSError`.